### PR TITLE
Addinng AMI creating script adapted from scylla

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

File: aws/ami/build_ami.sh [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+AMI=ami-ae7bfdb8
+REGION=us-east-1
+SSH_USERNAME=centos
+
+if [ $LOCALRPM -eq 1 ]; then
+    # TODO move to new repo
+    # REPO=`./scripts/scylla_current_repo --target centos`
+    INSTALL_ARGS="$INSTALL_ARGS --localrpm"
+
+    SCYLLA_VERSION=$(get_version_from_rpm $DIR/files/$PRODUCT-server*.x86_64.rpm)
```
Comment (syuu1228) [inline aws/ami/build_ami.sh]: We need to check *.rpm file existence.
When I tried ./aws/ami/build_ami.sh --localrpm, I mistakenly copied *.rpm to another directory, the script just proceed to build AMI and failed.

**Comment by syuu1228** [comment]
--repo didn't work for me:
```
[syuu@monster scylla-cloud-image]$ ./aws/ami/build_ami.sh --repo https://s3.amazonaws.com/downloads.scylladb.com/rpm/centos/scylla-3.1.repo
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   396  100   396    0     0    510      0 --:--:-- --:--:-- --:--:--   509
Repository scylla-3.1 is listed more than once in the configuration
Repository scylla-generic-3.1 is listed more than once in the configuration
Scylla for Centos 31 - x86_64                     283  B/s | 341  B     00:01    
Errors during downloading metadata for repository 'scylla-3.1':
  - Status code: 404 for https://s3.amazonaws.com/downloads.scylladb.com/downloads/scylla/rpm/centos/scylladb-3.1/31/x86_64/repodata/repomd.xml (IP: 52.217.10.22)
Error: Failed to download metadata for repo 'scylla-3.1': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
Scylla for centos 31                              286  B/s | 341  B     00:01    
Errors during downloading metadata for repository 'scylla-generic-3.1':
  - Status code: 404 for https://s3.amazonaws.com/downloads.scylladb.com/downloads/scylla/rpm/centos/scylladb-3.1/31/noarch/repodata/repomd.xml (IP: 52.216.9.189)
Error: Failed to download metadata for repo 'scylla-generic-3.1': Cannot download repomd.xml: Cannot download repodata/repomd.xml: All mirrors were tried
Ignoring repositories: scylla-3.1, scylla-generic-3.1
Last metadata expiration check: 4:49:13 ago on Fri Jan 10 07:32:32 2020.
No package scylla available.
Exiting due to strict setting.
Error: No package scylla available.
```

File: aws/ami/build_ami.sh (hunk 1 above)
Comment (fruch) [reply aws/ami/build_ami.sh]: Will add a check

**Comment by fruch** [comment]
@syuu1228 this repo seems broken regardless, I'll look into it in Sunday

**Comment by penberg** [comment]
@fruch I am missing the background and motivation for this. Why do you need a new AMI script? Is this _the_ new AMI that folks have been working or is it just for cloud?

**Comment by penberg** [comment]
@fruch I think you can simplify much more: for example, remove the `is_debian` variants altogether. You could even create a `Dockerfile` that has all the _build-time_ dependencies available and remove them from the Bash script. Overall, I would encourage you to not use `build_rpm.sh` as too much of a base, because it's not very robust or easy to use.

**Comment by fruch** [comment]
@penberg the cloud might be confusing a bit here. this repo is meant for building images for cloud provider. And not tied with scylla-cloud directly.

First supporting AWS, it's built and RPM formally known as scylla-ami. and we decided to move all the ami creating also out of scylla core.

So everything is packed in the same repo.

Also I would like the docs on how to use and build into this repo, and deprecate the different wiki pages around this subject.

**Comment by fruch** [comment]
Dockerizing it, is a good idea. But for now I'll still build it every time. Until I'll open up a dockerhub project for it under scylladb

File: aws/ami/build_ami.sh (hunk 1 above)
Comment (fruch) [reply aws/ami/build_ami.sh]: added a check

**Comment by fruch** [comment]
@penberg @syuu1228 I've add script to run it via docker, your input would be appreciated.

**Pull request merged**